tests/tasks.bats

```diff
@@ -13,6 +13,14 @@ setup() {
     "$DAGGER" "do" -p ./tasks/pull/pull_auth.cue pull
 }
 
+@test "task: #Pull auth env" {
+    export DOCKERHUB_AUTH_USER=foo DOCKERHUB_AUTH_PASSWORD=bar
+    run "$DAGGER" "do" -p ./tasks/pull/pull_auth_env.cue pull
+    unset DOCKERHUB_AUTH_USER DOCKERHUB_AUTH_PASSWORD
```

Comment:
You don't need to unset these vars. Bats does it for you (each test is isolated).

Reply:
You marked as resolved but kept the line?